src/proto/generation.proto

```diff
@@ -219,6 +219,16 @@ message TransformType {
   }
 }
 
+message CAIParameters {
+  enum ModelMetadata {
+    METADATA_UNSPECIFIED = 0;
+    SIGN_WITH_ENGINE_ID = 1;
```

Comment:
`nitpick`: I think if you run this through buf lint, you might get some suggestions on the naming format of the enum values....

https://buf.build/docs/lint/usage/

https://www.notion.so/stabilityai/Protocol-Buffer-Development-Guidelines-94c6a5b9082d4aa6a2d6bf50b86fe0ac

Otherwise this looks great!

Reply:
Thanks for the suggestion! Modified my enum values to adhere to buf's suggestions.